### ISSUE TITLE
Update install command in "Developer version"

### ISSUE THREAD

**Issue by pllim** [radio-astro-tools/spectral-cube#337]
Under the [Developer version](http://spectral-cube.readthedocs.io/en/latest/installing.html#developer-version) section, this command

```
pip install https://github.com/radio-astro-tools/spectral-cube/archive/master.zip
```

should be changed to

```
pip install git+https://https://github.com/radio-astro-tools/spectral-cube.git
```
.

Otherwise, the version tagging is inaccurate, as follows:

```
$ pip install https://github.com/radio-astro-tools/spectral-cube/archive/master.zip
...
Successfully installed spectral-cube-0.3.3.dev0


$ pip install git+https://https://github.com/radio-astro-tools/spectral-cube.git
...
Successfully installed spectral-cube-0.3.3.dev1240


$ python setup.py install
...
Finished processing dependencies for spectral-cube==0.3.3.dev1240
```

**Comment by astrofrog** [comment]
Note that ``pip install git+https://github.com/radio-astro-tools/spectral-cube.git`` works too (without the double http). I'll open a PR for this.

**Comment by astrofrog** [comment]
Fixed directly in master in e96c60ea0f44210ed7b1d41b32a6f22030f7aa0f

**Comment by pllim** [comment]
> without the double http

Ops! That "double http" is a copy and paste typo. It should just be single, as you indicated. Thanks for catching that and fixing the doc!
